charts/admission-controller/templates/_helpers.tpl

```diff
@@ -376,3 +376,109 @@ an error if not.
 true
 {{- end }}
 {{- end }}
+
+{{- define "admissionController.webhookTemplate" }}
+webhooks:
+{{- if .Values.features.kspmAdmissionController}}
+- name: vac.secure.sysdig.com
+  namespaceSelector:
```

Comment:
Adding the namespaceSelector to prevent issues with the AC blocking itself, instead of the post-install hook approach. From k8s changelog, this will only work in Kubernetes 1.21 or higher, where the kubernetes.io/metadata.name was added. But this should be old enough.